Tasks/GoTool/gotool.ts

```diff
@@ -111,4 +110,15 @@ function setGoEnvironmentVariables(goRoot: string) {
     }
 }
 
+// This function is required to convert the version 1.10 to 1.10.0.
+// Because caching utility accept only sementic version,
+// which have patch number as well.
+function fixVersion(version: string): string {
+    let versionPart = version.split(".");
+    if(versionPart == null) {
+        return version.concat(".0");
```

Comment:
How will this work in the example you state in the comments above

Reply:
I think, by mistake I undo the change. It should be versionPart[2] == null

fixing it